ffmpeg_tools/validation.py

```diff
@@ -126,6 +132,7 @@ def validate_transcoding_params(
     dst_params,
     src_metadata,
     dst_muxer_info,
+    dst_audio_encoder_info,
```

Comment:
Here we have the same problem as with muxer_info. We need it optional.

Reply:
Sure but I would have to repeat bits of code from a similar fix in P5b/#23 and even then it would conflict with this PR so I've been waiting to see if my changes there would be accepted. I see that now P5b passes CI tests for I'll rebase this one on top of it and add the necessary adjustments.

Reply:
Done